README.md

```diff
@@ -19,8 +19,7 @@
 
 # liatrio-otel-collector
 
-A repository containing custom OpenTelemetry (otel) packages to be used when
-building and configuring custom collectors.
+A repository containing custom OpenTelemetry (otel) packages to be used when building and configuring custom collectors.
```

Comment:
If we're going to go ahead and reformat this to one line (though it did render as one line originally), let's update this to more accurately reflect the purpose of the repo. 

Something like this perhaps

```suggestion
The Liatrio OTEL Collector is an upstream distribution of the Open Telemetry collector, rebuilt with custom packages hosted within this repository.  These custom packages are by default targeted for downstream contribution to Open Telemetry; pursuant acceptance by the community.
```

Reply:
Woops.. I changed this by mistake. I didn't mean to merge the two into one, I thought it was one point on two lines 😅